### ISSUE TITLE
React Dom does not updates

### ISSUE THREAD

**Issue by raflores1** [fedosejev/react-essentials#33]
I must have done something wrong, When I make changes in app.js, these changes do not reflect on the index.html as I open it on the browser. The file only updates after I run gulp, so do I have to do that every time I update app.js?

**Comment by fedosejev** [comment]
@raflores1 yes, you need to run `gulp` every time you update your `app.js` file. Alternatively, you can use `gulp.watch` method. For example: https://github.com/gulpjs/gulp#sample-gulpfilejs

**Comment by raflores1** [comment]
Thank you for your prompt response.

Rafael A. Flores - President
T. 787.200.2739
rafael.flores@ensobusiness.com
www.ensobusiness.com
